### PR TITLE
API Review: PDF toolbar customization

### PR DESCRIPTION
**Pull request by SamuelQZQ**
This is a review for the new PDF toolbar customization API.

### CODE REVIEW

File: specs/HiddenPdfToolbarItems.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Background
+When open a PDF file in Edge, there will be a toolbar at the top. The toolbar provide the functionality like print, save and annotation.
```
Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:2]: Just a few minor text fixes
```suggestion
When opening a PDF file in Edge, there will be a toolbar at the top. The toolbar provides functionality like print, save and adding annotations.
```

File: specs/HiddenPdfToolbarItems.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+And this PDF viewer is also available in WebView2. And we provide the API to enable users to customization the PDF toolbar.
```
Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:3]:
```suggestion
This PDF viewer is also available in WebView2. And we provide this new API to enable end developers to customize the PDF toolbar.
```

File: specs/HiddenPdfToolbarItems.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+
+# Description
+We add a new `HiddenPdfToolbarItems` property in `CoreWebView2Settings`. This API allows users to hide buttons on the PDF toolbar.
```
Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:7]:
```suggestion
We add a new `HiddenPdfToolbarItems` property in `CoreWebView2Settings`. This API allows end developers to hide buttons on the PDF toolbar. 
```

File: specs/HiddenPdfToolbarItems.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+  COREWEBVIEW2_PDF_TOOLBAR_ITEM_SAVE_AS  = 0x0004,
+
+
+} COREWEBVIEW2_PDF_TOOLBAR_ITEM;
+cpp_quote("DEFINE_ENUM_FLAG_OPERATORS(COREWEBVIEW2_PDF_TOOLBAR_ITEM);")
+
+[uuid(183e7052-1d03-43a0-ab99-98e043b66b39), object, pointer_default(unique)]
+interface ICoreWebView2Settings6 : ICoreWebView2Settings5 {
+  /// `HiddenPdfToolbarItems` is used to customize the PDF toolbar items. By default, it displays all of the items.
```
Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:88]:
```suggestion
  /// `HiddenPdfToolbarItems` is used to customize the PDF toolbar items. By default, it is  COREWEBVIEW2_PDF_TOOLBAR_ITEM_NONE and so it displays all of the items.
```

Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:88]: If I recall from a different conversation this setting applies to more than just this CoreWebView2, correct? Does it apply to all CoreWebView2s using the same browser process and profile? If so, we need to specify this in the documentation here. 
```suggestion
  /// `HiddenPdfToolbarItems` is used to customize the PDF toolbar items. By default, it displays all of the items.
  /// Changes to this property apply to all CoreWebView2s in the same environment and using the same profile.
```

File: specs/HiddenPdfToolbarItems.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  /// \snippet SettingsComponent.cpp ToggleHidePdfToolbarItems
```
Comment (david-risney) [inline specs/HiddenPdfToolbarItems.md:89]: Changes to this setting only apply on the next navigation, correct?
```suggestion
  /// Changes to this setting apply only after the next navigation.
  /// \snippet SettingsComponent.cpp ToggleHidePdfToolbarItems
```

**Review by david-risney — CHANGES_REQUESTED** [review]

File: specs/HiddenPdfToolbarItems.md (hunk 1 above)
Comment (SamuelQZQ) [reply specs/HiddenPdfToolbarItems.md]: Some stupid faults. I will be more careful with syntax.

**Review by david-risney — APPROVED** [review]
This looks good thanks! Please complete the PR and open the next PR for this spec to master, let me know and I will create an API review meeting for it. Thanks!

**Pull request merged**